src/common/updating/notifications.cpp

```diff
@@ -24,122 +24,42 @@ namespace updating
+            if (openURL)
+            {
+                winrt::Windows::System::Launcher::LaunchUriAsync(info.release_page_uri).get();
+            }
         }
 
         void show_install_error(const updating::new_version_download_info& info, const notifications::strings& strings)
         {
-            remove_toasts_by_tag(UPDATING_PROCESS_TOAST_TAG);
-
-            toast_params toast_params{ UPDATING_PROCESS_TOAST_TAG, false };
             std::wstring contents = strings.GITHUB_NEW_VERSION_DOWNLOAD_INSTALL_ERROR;
             contents += L'\n';
             contents += current_version_to_next_version(info);
-            show_toast_with_activations(std::move(contents),
-                                        strings.TOAST_TITLE,
-                                        {},
-                                        { link_button{ strings.GITHUB_NEW_VERSION_VISIT, info.release_page_uri.ToString().c_str() } },
-                                        std::move(toast_params));
+            MessageBoxW(nullptr, contents.c_str(), strings.NOTIFICATION_TITLE.c_str(), MB_OK | MB_ICONERROR);
         }
 
-        void show_version_ready(const updating::new_version_download_info& info, const notifications::strings& strings)
+        bool show_confirm_update(const updating::new_version_download_info& info, const notifications::strings& strings)
         {
-            remove_toasts_by_tag(UPDATING_PROCESS_TOAST_TAG);
-
-            toast_params toast_params{ UPDATING_PROCESS_TOAST_TAG, false };
             std::wstring new_version_ready{ strings.GITHUB_NEW_VERSION_READY_TO_INSTALL };
             new_version_ready += L'\n';
             new_version_ready += current_version_to_next_version(info);
 
-            show_toast_with_activations(std::move(new_version_ready),
-                                        strings.TOAST_TITLE,
-                                        {},
-                                        { link_button{ strings.GITHUB_NEW_VERSION_UPDATE_NOW,
-                                                       L"powertoys://update_now/" + info.installer_filename },
-                                          link_button{ strings.GITHUB_NEW_VERSION_UPDATE_AFTER_RESTART,
-                                                       L"powertoys://schedule_update/" + info.installer_filename },
-                                          snooze_button{
-                                              strings.GITHUB_NEW_VERSION_SNOOZE_TITLE,
-                                              { { strings.GITHUB_NEW_VERSION_UPDATE_SNOOZE_1D, 24 * 60 },
-                                                { strings.GITHUB_NEW_VERSION_UPDATE_SNOOZE_5D, 120 * 60 } },
-                                              strings.SNOOZE_BUTTON } },
-                                        std::move(toast_params));
+            return IDYES == MessageBoxW(nullptr, new_version_ready.c_str(), strings.NOTIFICATION_TITLE.c_str(), MB_ICONQUESTION | MB_YESNO);
```

Comment:
We need to add an explicit question, since right now the dialog shows:
```
An update to PowerToys is ready to install.
v0.33.0 -> 0.35.0
```

It should be:
```
An update to PowerToys is ready to install.
v0.33.0 -> 0.35.0
Do you want to update now?
```

Also: at this point the installer has already been already downloaded?